examples/tokenvm/avalanche-ops.aws.md

```diff
@@ -0,0 +1,249 @@
+
+```bash
+# first make sure you have access to the AWS credential
+aws sts get-caller-identity
+
+# "avalancheup-aws apply" would not work...
+# An error occurred (ExpiredToken) when calling the GetCallerIdentity operation: The security token included in the request is expired
+
+# "avalancheup-aws apply" will work with the following credential
+# {
+#   "UserId": "...:abc@abc.com",
+#   "Account": "...",
+#   "Arn": "arn:aws:sts::....:assumed-role/name/abc@abc.com"
+# }
+```
+
+```bash
+# run the following to create resources
+vi /home/ubuntu/aops-custom-****-***.yaml
+
+avalancheup-aws apply \
+--spec-file-path /home/ubuntu/aops-custom-****-***.yaml
+
+# run the following to delete resources
+avalancheup-aws delete \
+--delete-cloudwatch-log-group \
+--delete-s3-objects \
+--delete-ebs-volumes \
+--delete-elastic-ips \
+--spec-file-path /home/ubuntu/aops-custom-****-***.yaml
+```
+
+That is, `apply` creates AWS resources, whereas `delete` destroys after testing is done.
+
+*SECURITY*: By default, the SSH and HTTP ports are open to public. Once you complete provisioning the nodes, go to EC2 security group to restrict the inbound rules to your IP.
+
+*NOTE*: In rare cases, you may encounter [aws-sdk-rust#611](https://github.com/awslabs/aws-sdk-rust/issues/611) where AWS SDK call hangs, which blocks node bootstraps. If a node takes too long to start, connect to that instance (e..g, use SSM sesson from your AWS console), and restart the agent with the command `sudo systemctl restart avalanched-aws`.
+
+### Step 4
+
+Now that the network and nodes are up, let's install two subnets, each of which runs its own TokenVM. Note that the above, successful `avalancheup-aws apply` run will output the following commands as an example, which you can customize depending on the VMs:
+
+```bash
+# EXAMPLE: write subnet config
+avalancheup-aws subnet-config \
+--log-level=info \
+--proposer-min-block-delay 250000000 \
+--file-path /tmp/subnet-config.json
+
+# EXAMPLE: install subnet + custom blockchain in all nodes
+avalancheup-aws install-subnet-chain \
+--log-level info \
+--region us-west-2 \
+--s3-bucket avalanche-ops-***-us-west-2 \
+--s3-key-prefix aops-custom-***/install-subnet-chain \
+--ssm-doc aops-custom-***-ssm-install-subnet-chain \
+--chain-rpc-url http://52.88.8.107:9650 \
+...
+```
+
+For instance, use the following to **deploy two TokenVM subnets and create separate TokenVM chains** (similar to [scripts/run.sh](./scripts/run.sh)):
+
+```bash
+avalancheup-aws subnet-config \
+--log-level=info \
+--proposer-min-block-delay 0 \
+--file-path /tmp/subnet-config.json
+
+cat <<EOF > /tmp/allocations.json
+[{"address":"token1rvzhmceq997zntgvravfagsks6w0ryud3rylh4cdvayry0dl97nsjzf3yp", "balance":1000000000000}]
+EOF
+rm -f /tmp/tokenvm-genesis.json
+/tmp/token-cli genesis generate /tmp/allocations.json \
+--genesis-file /tmp/tokenvm-genesis.json
+cat /tmp/tokenvm-genesis.json
+
+cat <<EOF > /tmp/tokenvm-chain-config.json
```

Comment:
Please update this with the new fields I added: https://github.com/ava-labs/hypersdk/blob/6fce12a85e9b618afbf446d80181886ecfd5507e/examples/tokenvm/scripts/run.sh#L115-L124